src/fprime/fbuild/cli.py

```diff
@@ -57,11 +57,11 @@ def run_fbuild_cli(
         toolchain = build.find_toolchain()
         print(f"[INFO] Generating build directory at: {build.build_dir}")
         print(f"[INFO] Using toolchain file {toolchain} for platform {parsed.platform}")
-        if parsed.ut and not parsed.disable_sanitizers:
+        if parsed.disable_sanitizers:
```

Comment:
This line is the only place in the system (currently) where sanitizers do get enabled. Without it, sanitizers will always be disabled unless the `ENABLE_SANITIZERS_*` flags are passed to CMake.

Is this the intended behavior? It feels to me that sanitizers on UTs are a good practice.

Not sure why nasa/fprime#2678 fails though...

Reply:
Oh I should have read nasa/fprime#2678 prior to posting this lol

Are you intending to make a PR to F´ to have `SANITIZERS_* = BUILD_TESTING`

Reply:
My intent was to set sanitizers in CMake in F´, yes.